content/PreparedStatement/PreparedStatement.q.md

```diff
@@ -0,0 +1,1707 @@
+# Prepared Statement
+
++++
+
+### Part 1: Sample Code Analysis
+
+:::
+
+/// type=REPL, readonly=true, filename=[connection.php,student.php], init=[commands/PreparedStatement/InsertValuesBerryVikkiIntoStudentTable.sql]
+
+```php
+// connection.php
+<?php
+    $host = 'localhost';
+    $db = 'LibraryDB';
+    $port = '5432';
+    $username = 'postgres';
```

Comment:
can we use a different username, `postgres`  user is a super user by default and we don't want to change it.

Reply:
Got it manager. Let me go ahead and make changes to the username. Thanks.